src/ch01/dice_game_vrf.md

```diff
@@ -0,0 +1,11 @@
+# Dice Game using Pragma VRF
+
+This code provides an implementation of a Dice Game contract that utilizes a [Pragma Verifiable Random Function (VRF)](https://docs.pragma.build/Resources/Cairo%201/randomness/randomness) to generate random numbers on-chain.
+
```

Comment:
What do you think about explaining a bit more about randomness in the context of blockchain before jumping directly in the implementation? Especially to outline where the entropy comes from and potential risks with it. You can then just link to [Pragma - Randomness](https://docs.pragma.build/Resources/Cairo%201/randomness/randomness#how-randomness-is-generated)

Reply:
Entropy sources:
- Blockchain: block hashes, timestamps, ...
- External Sources: Oracles

Risks: Predictability/Manipulation/Centralization-trust

Reply:
Alright. I'll make an update to this week